Formula/mypy.rb

```diff
@@ -2,8 +2,8 @@ class Mypy < Formula
   desc "Experimental optional static type checker for Python"
   homepage "http://www.mypy-lang.org/"
   url "https://github.com/JukkaL/mypy.git",
-    :tag => "v0.4.2",
-    :revision => "b8a2df91707bc3d5c9c3677c8017b614e41d017a"
+      :tag => "v0.4.3",
```

Comment:
Given all the ways we use indent in Homebrew, even within formulae, I'm not sure there's ever going to be a simple, programmatic way to enforce it to be honest... happy to be proven wrong in time though.